src/signals/incident/definitions/wizard-step-2-vulaan/wonen.ts

```diff
@@ -778,6 +778,200 @@ const vakantieverhuur = {
+  },
+  extra_wonen_verhuurderschap_discriminatie_anders: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_discriminatie: 'anders',
+      },
+      label: 'Een andere reden, namelijk:',
+      shortLabel: 'Andere reden',
+      pathMerge: 'extra_properties',
+      type: 'url',
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.TextInput,
+  },
+  extra_wonen_verhuurderschap_discriminatie_bewijs: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_onderwerp: 'discriminatie',
+      },
+      label: 'Aanvullende informatie:',
+      shortLabel: 'Bewijs discriminatie',
+      pathMerge: 'extra_properties',
+      values: {
+        bewijs:
+          'Kunt u de discriminatie tijdens het verhuurproces aantonen door bijvoorbeeld een afwijzing van de woning of de selectiecriteria voor het verkrijgen van de woning? (Zo ja, voeg deze stukken toe aan de bijlage)',
+      },
+    },
+    render: QuestionFieldType.CheckboxInput,
+  },
+
+  // Intimidatie
+  extra_wonen_verhuurderschap_intimidatie: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_onderwerp: 'intimidatie',
+      },
+      label:
+        'Bent u bijvoorbeeld tijdens het verhuurproces door lichamelijk of psychisch geweld of bedreiging met geweld gedwongen om iets te doen of juist iets niet te doen?',
+      shortLabel: 'Intimidatie',
+      pathMerge: 'extra_properties',
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.TextInput,
+  },
+
+  // Huurcontract
+  extra_wonen_verhuurderschap_huurcontract: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_onderwerp: 'huurcontract',
+      },
+      label: 'Vink aan wat van toepassing is:',
+      shortLabel: 'Huurcontract',
+      pathMerge: 'extra_properties',
+      values: {
+        bezit_huurcontract: 'Bent u in het bezit van een huurcontract?',
+        rechten_plichten:
+          'Zijn uw rechten en plichten schriftelijk vastgesteld? (Zo ja, voeg deze stukken toe aan de bijlage)',
+        borg: 'Is de hoogte van de borg op schrift vastgesteld en is de termijn van vordering van de borg bij beëindiging van de huurovereenkomst op schrift vastgesteld? (Zo ja, voeg deze stukken toe aan de bijlage)',
+      },
+    },
+    render: QuestionFieldType.CheckboxInput,
+  },
+
+  // Dubbele bemiddelingskosten
+  extra_wonen_verhuurderschap_bemiddelingskosten: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_onderwerp: 'bemiddelingskosten',
+      },
+      label: 'Vink aan wat van toepassing is:',
+      shortLabel: 'Bemiddelingskosten',
+      pathMerge: 'extra_properties',
+      values: {
+        extra_kosten:
+          'Heeft u naast de huur en de borg nog meer moeten betalen aan de bemiddelaar of makelaar? (Zo ja, voeg deze stukken toe aan de bijlage)',
+      },
+    },
+    render: QuestionFieldType.CheckboxInput,
```

Comment:
Beetje vreemd om een checkbox te hebben terwijl er maar 1 keuze is. Zou een regel tekst met uitleg niet beter zijn?